lib/sshnp.dart

```diff
@@ -299,13 +305,14 @@ class SSHNP {
     }
   }
 
-
   /// Look up the user name ... we expect a key to have been shared with us by
   /// sshnpd. Let's say we are @human running sshnp, and @daemon is running
   /// sshnpd, then we expect a key to have been shared whose ID is
   /// @human:username.device.sshnp@daemon
+  /// fetches remote user name if not provided through class constructor
```

Comment:
Replace with "Is not called if remoteUserName was set via constructor"

Reply:
Done